shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -253,8 +256,17 @@ private void connectSurfaceToRenderer() {
       throw new IllegalStateException(
           "connectSurfaceToRenderer() should only be called when flutterRenderer and getHolder() are non-null.");
     }
-
-    flutterRenderer.startRenderingToSurface(getHolder().getSurface());
+    // When connecting the surface to the renderer, it's possible that the surface is currently
+    // paused. For instance, when a platform view is displayed, the current FlutterSurfaceView
+    // is paused, and rendering continues in a FlutterImageView buffer while the platform view
+    // is displayed.
+    //
+    // startRenderingToSurface may stop rendering to this surface if it believes it already started
```

Comment:
Nit, the wording here is a bit confusing? Not sure if can be better said.

Reply:
agreed. Done